bundles/org.openhab.binding.cul.max/README.md

```diff
@@ -0,0 +1,204 @@
+
+## Binding Configuration
+
+There are no binding wide settings as all configuration settings.
+When using a serial port, you may need to add `-Dgnu.io.rxtx.SerialPorts=/dev/ttyACM0` in your server startup.  Please consult the [forum](https://community.openhab.org) for the latest information.
+
+## Thing Configuration
+
+### CUL Bridge Configuration
+| Property | Default | Required | Description |
+|----------|---------|:--------:|-------------|
+| device   |         |   Yes    | in the form `<device>`, where `<device>` is a local serial port eg `/dev/ttyACM0` |
+| baudrate |         |   No     | one of 75, 110, 300, 1200, 2400, 4800, 9600, 19200, **38400**, 57600, 115200 |
+| parity   |         |   No     | one of EVEN, ODD, MARK, **NONE**, SPACE |
+
+When using a serial port, you may need to add `-Dgnu.io.rxtx.SerialPorts=/dev/ttyACM0` in your server startup.  Please consult the [forum](https://community.openhab.org) for the latest information.
+
+### CUN Bridge Configuration
+| Property      | Default | Required | Description |
+|---------------|---------|:--------:|-------------|
+| networkPath   |         |   Yes    | in the form `<device>`, where `<device>` is `<host>:<port>`, where `<host>` is the host name or IP address and `<port>` is the port number. If no `<port>` is provided the default `2323` is assumed |
+
+### MAX! CUL Bridge Configuration
+| Property      | Default | Required | Description |
+|---------------|---------|:--------:|-------------|
+| timezone | Europe/London | No | set timezone you want the units to be set to |
+
+### Linking devices
+| Property      | Default | Required | Description |
+|---------------|---------|:--------:|-------------|
+| associate|  | No | Comma seperated list of serials|
+
+### All devices
+| Property      | Default | Required | Description |
+|---------------|---------|:--------:|-------------|
+| serial        |         |   Yes    | eg. KEQ12345|
+| rfAddress     |         |   Yes    | eg. 071234  |
+
+
+### Additional Properties if available
+
+* comfort - the defined 'comfort' temperature (default 21.0)
+* eco - the defined eco setback temperature (default 17.0)
+* max - maximum temperature that can be set on the thermostat (default 30.5, which is the maximum value and corresponds to "open valve")
+* min - minimum temperature that can be set on the thermostat (default 4.5, which is the minimum value and corresponds to "closed valve")
+* windowOpenDetectTemp - set point in the event that a window open event is triggered by a shutter, if set to 4.5, this function is deactivated.
+* windowOpenDetectTime - Rounded down to the nearest 5 minutes. (default is 0)
+* measurement offset - offset applied to measure temperature (range is -3.5 to +3.5) - default is 0.0
+* weekprofile - eg. `Mon;17,17-30,21,23-00,17;Tue;17,17-30,21,23-00,17;Wen;17,17-30,21,23-00,17;Thu;17,17-30,21,23-00,17;Fri;17,17-30,21,23-30,17;Sat;17,09-00,21,23-00,17;Sun;17,09-00,21,23-00,17`

```

Comment:
These seem rather configuration parameters than properties. Can you mention to what they are apply? You could put them into a table.